pkg/platform/runtime/runtime.go

```diff
@@ -152,6 +152,13 @@ func (r *Runtime) Env(inherit bool, useExecutors bool) (map[string]string, error
 	env := envDef.GetEnv(inherit)
 
 	execDir := filepath.Clean(setup.ExecDir(r.target.Dir()))
+	if bashifyPaths {
+		var err error
+		execDir, err = osutils.BashifyPath(execDir)
+		if err != nil {
+			return nil, errs.Wrap(err, "Could not bashify path: %s", execDir)
+		}
+	}
```

Comment:
Isn't this redundant? Ultimately this env info will be handled by subshell. I feel like the bashify stuff should probably be centralized in the subshell package.

Reply:
No, we need to catch it here. This is where we "// Override PATH entry with exec path", so that exec path needs to be explicitly bashified if needed. There is no notion of subshell at this point.

Reply:
I understand, what I meant is that the env that this returns is ultimately used through the subshell package. If not; can you point me at the code that's causing bash incompatibility that the subshell package is not addressing?

Reply:
Never mind. It looks like setting up executors appears to use an env without subshell, so we probably still need to bashify here in runtime env setup: https://github.com/ActiveState/cli/blob/5080107ce157556e7bbffb0314afdf9a8185896c/internal/globaldefault/default.go#L75

Reply:
For executors it shouldn't matter, they're executables. They should not care what shell invoked them. 

> Are you expecting the subshell to iterate through all of its environment variables, look for paths that should be bashified, and then do so? 

I think so. Open to suggestions but this seems like the way to do it.

> but I thought it'd be less error-prone to bashify paths at their source rather than downstream.

I feel like this is more error prone as now everything dealing with this data is dealing with bashified paths, rather than JUST the component that cares about this (ie. subshell).

Reply:
Subshells are now responsible for bashifying their environment variables. The bash subshell now does this.